pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[tool.poetry]
+name = "supervision"
+version = "0.12.0"
```

Comment:
Until now, we stored our project version in the `supervision/__init__.py` file. And we have dynamically added it to `setup.py`. Is it possible with `pyproject.toml`?

Reply:
I added 

import importlib.metadata as importlib_metadata
__version__ = importlib_metadata.version(__package__)

So __version__ become dynamic and we going to only change toml. I think that would be okay too ?

Reply:
@onuralpszr @SkalskiP I think there is a way to make it dynamically based on git commit and tag version. I haven't done it but we had it done in my old company. `project.toml` or `setup.py` `version` can be change as a part of `DevOps`.

Reply:
> @onuralpszr @SkalskiP I think there is a way to make it dynamically based on git commit and tag version. I haven't done it but we had it done in my old company. `project.toml` or `setup.py` `version` can be change as a part of `DevOps`.

I know how to handle via github action as in "DevOps" term but for now, I just want to make a clean migration and we just need to change version toml file so it is easy to handle that. Maybe in later on we can talk about how to make full automation via tags. For now I will just keep like this.

Reply:
@onuralpszr okey, so now we will be doing stuff the other way around? Instead importing `__version__` to `setup.py` we will import `__version__` from `project.toml`?

@hardikdava Cool idea! But let's keep versions bumping manual for now. We are doing so many changes from a DevOps perspective it makes me stressed... Iterative changes 🙏🏻